net-firewall/iptables/iptables-1.8.7.ebuild

```diff
@@ -43,8 +43,6 @@ BDEPEND="${BUILD_DEPEND}
 RDEPEND="${COMMON_DEPEND}
 	${BUILD_DEPEND}
 	nftables? ( net-misc/ethertypes )
-	!<net-firewall/ebtables-2.0.11-r1
-	!<net-firewall/arptables-0.0.5-r1
```

Comment:
A simple comment to say why those are needed, like "not needed as nftables is disabled by default".

Reply:
Is there a reason for dropping these blocks?

Reply:
Sounds good.